src/LevelAgreement.php

```diff
@@ -110,7 +110,7 @@ public function defineTabs($options = [])
         $this->addDefaultFormTab($ong);
         $this->addStandardTab(static::$levelclass, $ong, $options);
         $this->addStandardTab('Rule', $ong, $options);
-        $this->addStandardTab('Ticket', $ong, $options);
+        $this->addStandardTab('Item_Ticket', $ong, $options);
```

Comment:
I am not sure to understand. `Item_Ticket` is supposed to show items related to a ticket. Why using it in a service level context?

Reply:
The Item_Ticket class simply calls `Ticket::showListForItem` when displaying the tab. The method in Ticket handles more than assets; it handles users, sla, ola, suppliers, and groups too.
Defining a tab for `Ticket::class`, you get a deprecation warning telling you to use `Item_Ticket::class` instead.